### PR TITLE
chore: update schema after sql launch

### PR DESCRIPTION
**Pull request by jimniels**
Now that #766 has shipped and the data migration has run successfully, we can update the schema to properly reflect the new expected state of the database. This includes:

- `ownerTeamId` and `ownerTeam` go from being optional to required on `File`
- `activated` on a `Team` gets dropped, as we no longer need (or reference anywhere) this value (it's what we used to use when a user would create a team via stripe but bail before completing the task).

This gets all our types working correctly in the API and matching expectations we have about the data.

### CODE REVIEW

File: quadratic-client/src/dashboard/components/CustomRadixIcons.tsx [hunk 1]
```diff
@@ -37,14 +37,14 @@ export const SharedWithMeIcon = React.forwardRef<SVGSVGElement, IconProps>(
...
+          fillRule="evenodd"
+          clipRule="evenodd"
           d="M3.5 5.00006C3.22386 5.00006 3 5.22392 3 5.50006V11.5001C3 11.7762 3.22386 12.0001 3.5 12.0001H11.5C11.7761 12.0001 12 11.7762 12 11.5001V5.50006C12 5.22392 11.7761 5.00006 11.5 5.00006H10.25C9.97386 5.00006 9.75 4.7762 9.75 4.50006C9.75 4.22392 9.97386 4.00006 10.25 4.00006H11.5C12.3284 4.00006 13 4.67163 13 5.50006V11.5001C13 12.3285 12.3284 13.0001 11.5 13.0001H3.5C2.67157 13.0001 2 12.3285 2 11.5001V5.50006C2 4.67163 2.67157 4.00006 3.5 4.00006H4.75C5.02614 4.00006 5.25 4.22392 5.25 4.50006C5.25 4.7762 5.02614 5.00006 4.75 5.00006H3.5Z"
           fill={color}
         />
         <path
-          fill-rule="evenodd"
-          clip-rule="evenodd"
+          fillRule="evenodd"
+          clipRule="evenodd"
```
Comment (jimniels) [inline quadratic-client/src/dashboard/components/CustomRadixIcons.tsx:47]: These aren't schema-related but noticed warnings about it from React so fixed here

**Review by davidkircos — APPROVED** [review]

**Comment by davidkircos** [comment]
API tests fail on CI

**Comment by davidkircos** [comment]
Going to keep this one against `main` and merge.

File: quadratic-api/src/stripe/stripe.ts [hunk 2]
```diff
@@ -158,7 +158,7 @@ const updateTeamStatus = async (
   await dbClient.team.update({
     where: { stripeCustomerId: customerId },
     data: {
-      activated: true, // activate the team
+      // activated: true, // activate the team
```
Comment (ddimaria) [inline quadratic-api/src/stripe/stripe.ts:161]: Remove?

**Review by ddimaria — APPROVED** [review]

**Review by davidkircos — APPROVED** [review]

**Pull request merged**